specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/HybridLinkListByInstance.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "managedInstanceName": "testcl",
+    "api-version": "2020-02-02-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "targetDatabase": "testdb",
+              "sourceEndpoint": "TCP://SERVER:7022",
+              "replicationMode": "Async",
+              "distributedAvailabilityGroupId": "6bc05a51-aa36-a196-09bd-481d7a0973c0",
+              "sourceReplicaId": "543dd519-7585-faff-6ad2-11fb826d4f4d",
+              "targetReplicaId": "7e218aba-0a53-6231-be09-895d99f96bf2"
+            },
+            "id": "/subscriptions/f2669dff-5f08-45dd-b857-b2a60b72cdc9/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testcl/hybridLink/dag",
+            "name": "dag",
+            "type": "Microsoft.Sql/managedInstances/hybridLink"
+          }
+        ]
+      }
```

Comment:
this would have nextlink right?

Reply:
Yeah, I'll add more items in this example.